scripts/tap_to_text.pl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/perl
+#
+# Write a textual summary of a TAP file
+
+use strict;
+use warnings FATAL => 'all';
+
+use TAP::Parser;
+
+my $RED = "\e[31m";
+my $RESET_FG = "\e[39m";
+
+my $tap_file = $ARGV[0];
+
+my $parser = TAP::Parser->new( { source => $tap_file } );
+my $in_error = 0;
+my $expected_fail = 0;
+
+while ( my $result = $parser->next ) {
+   if ( $result->is_test ) {
+      # conclude any previous error block
+      if( $in_error ) {
+         print "\n";
+      }
+
+      $in_error = 0;
+
+      if ( not $result->is_ok ) {
+         $in_error = 1;
+
+         my $number = $result->number;
+         my $description = $result->description;
+
+         print "${RED}FAILURE:$RESET_FG #$number: $description\n";
```

Comment:
I wonder if we should look for `GITHUB_ACTIONS` being [set in the environment](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables), and if so, log the error in a [format that it will detect](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message)?

E.g., `print ("::error::#$number: $description\n");`

Reply:
Ahh I looked for a document like that, and failed to find one.

I think it makes sense for this to be GHA-specific, rather that having magic based on the environment. I'll have a go at this.